src/themes/material/templates/journal/authors.html

```diff
@@ -0,0 +1,43 @@
+{% extends "core/base.html" %}
+{% load static from staticfiles %}
+
+{% block title %}Authors{% endblock title %}
+{% block page_title %}Authors{% endblock page_title %}
+
+{% block body %}
+
+    <h3>Authors</h3>
+    <div class="row">
+       
+        {% for author in author_list %}
+            <div class="col s12 m3">
+                <div class="card editorial-card">
+                    <div class="card-image">
+                         <img src="{% if author.profile_image %}{{ author.profile_image.url }}{% else %}{% static "common/img/icons/users.png" %}{% endif %}"
+                         alt="Card image cap">
+                    </div>
+                    <div class="card-content" style="min-height: 190px;">
+                         <span class="card-title small-card-title">{{ author.full_name }}</span>
+                        <p>
+                            <small>{{ author.affiliation }}</small>
+                        </p>
+                        {% if author.enable_public_profile %}
+                            <p>
+                                <small><a href="{% url 'core_public_profile' author.uuid %}">View Profile</a>
+                                </small>
+                            </p>
+                        {% endif %}
+
+                        <p>
+                           {% include "elements/journal/editorial_social_content.html" with user=author %}
+                        </p>
+
+                    </div>
+                </div>
+            </div>
+                       
+        {% endfor %}
+
+    </div>
+
+{% endblock body %}
```

Comment:
Missing new line here (They are missing throughout most of the code base but we are trying to add them for new files and whenever we touch current files)